python/semantic_kernel/memory/weaviate_memory_store.py

```diff
@@ -0,0 +1,314 @@
+import asyncio
+from dataclasses import dataclass
+from logging import Logger
+from typing import List, Optional, Tuple
+
+import numpy as np
+import weaviate
+from weaviate.embedded import EmbeddedOptions
+
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+from semantic_kernel.utils.null_logger import NullLogger
+
+SCHEMA = {
```

Comment:
I'd like to see the weaviate_memory_store found under the semantic_kernel/connectors/memory namespace.